python/server/ServerWrapper.cpp

```diff
@@ -905,6 +905,25 @@ namespace {
         return field->ID();
     }
 
+    // Return all systems within \p jumps of \p sys_ids
+    list SystemsWithinJumps(size_t jumps, list sys_ids) {
+        list py_systems;
+        boost::python::stl_input_iterator<int> end;
+
+        std::vector<int> systems;
+
+        for (boost::python::stl_input_iterator<int> id(sys_ids); id != end; ++id) {
+            systems.push_back(*id);
+        }
+
+        auto near = GetPathfinder()->WithinJumps(jumps, systems);
```

Comment:
Could you provide suggestions?

I've found that no matter what names I choose someone finds them too terse, too verbose, too cryptic or all of the above.